### DISCUSSION TITLE
Mac Aggregate Device does not allow for simultaneous recording of system audio and Yeti mic

### DISCUSSION THREAD

**Discussion by Stoopac** [ExistentialAudio/BlackHole#716] [General]
Hello everyone!

I can't seem to get the aggregate device to capture both the audio from the system and the audio from the microphone at the same time (which is what I really want to do). 

Any suggestions on how to get this done?

![Screenshot 2023-03-30 at 5 26 49 PM](https://user-images.githubusercontent.com/112725872/228986678-db2400a3-f141-4e3c-ac17-c92c0bce20e1.png)

**Comment by devinroth** [comment]
How are you recording? You need to record all channels as an aggregate does not mix them.

Devin

**Reply by Stoopac** [reply]
I'm trying to stream via Streamyard. The only thing I've been able to have work is having all the sound from the system on the left channel and all the audio from the USB on the right channel.

Thank you so much for helping out!

Here is what is happening via Garageband.
![Screenshot 2023-03-30 at 5 31 36 PM](https://user-images.githubusercontent.com/112725872/228987219-0462eeb5-2259-41db-96b9-0defdd4123bc.png)

**Comment by devinroth** [comment]
Garage band only is able to record 2 channels. Your aggregate has 5.

Devin

**Comment by Stoopac** [comment]
Any suggestions? I found a workaround, but it ain’t elegant. 🤣

**Comment by devinroth** [comment]
First off, I use Logic. 2nd, I’m developing an app called Dipper which would solve this problem for you. If you’re on Ventura you can test out the alpha build. The builds can be found on releases.

https://discord.gg/pmWKPcxhgt

Devin

**Comment by Stoopac** [comment]
Wow! Thank you! Looking forward to testing this out!
